src/backend/utils/error/debugutils.c

```diff
@@ -233,6 +245,12 @@ void dump_mt_bind(MemTupleBinding *mt_bind, const char *fname)
     FILE *ofile = fopen(fname, "w+");
     int i;
 
+    if (ofile == NULL)
+    {
+        fprintf(stderr, "dump_mt_bind: can't open \"%s\" for output", fname);
```

Comment:
I suggest to use __func__ to print function name, because someday the function name may be changed and it may cause to difference between the name and the string

Reply:
reworked to elog